infra/copyright/copyright-checker.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {EOL} from 'os';
+import {glob} from 'glob';
+import {readFileSync, writeFileSync} from 'fs';
+
+const copyrightRegex = new RegExp('Copyright \\(c\\) [0-9]{4} Samsung Electronics Co\\., Ltd\\. All Rights Reserved');
+
+const fileFormatsToCheck = "ts,js";
```

Comment:
(This question could be awkward since I have no background knowledge about copyright and licenses.) Don't we need to announce about the copyright or licences to `html` or `css` files?

Reply:
I missed this...! Thanks for pointing out :)